modules/pam_faillock/faillock.8.xml

```diff
@@ -55,14 +55,31 @@
 
     <title>OPTIONS</title>
          <variablelist>
+            <varlistentry>
+              <term>
+                <option>--conf <replaceable>/path/to/config-file</replaceable></option>
+              </term>
+              <listitem>
+                <para>
+                  The directory where the configuration file is located. The
```

Comment:
This is not a directory. It's just the file. `The path to the configuration file. The...`

Reply:
Fixed